src/main/java/edu/ucla/library/services/utility/Constants.java

```diff
@@ -0,0 +1,32 @@
+
+package edu.ucla.library.services.utility;
+
+public final class Constants {
```

Comment:
Could also name this ExitCodes, or something, to be more specific since that's the only sort of constant it has in it. Not a big deal, just a suggestion. Also, just an observation (and not a big deal), but some of the comments/javadocs start with capital letters and some don't.